### DISCUSSION TITLE
升级3.17.0的VNote，待办显示错误

### DISCUSSION THREAD

**Discussion by minli04g** [tamlok/vnote#2518] [Q&A]
Windows10 系统。 
![image](https://github.com/vnotex/vnote/assets/18209024/3226e335-1ca4-46b0-ac33-17bb2268d0c8)

**Comment by tamlok** [comment]
看来是XSS解决方案的副作用。

右上角菜单，编辑用户配置文件，把下面protect_from_xss设置为false：

![image](https://github.com/vnotex/vnote/assets/5763157/48ca455a-685d-4e29-a5a6-6ec67ed50d6a)

**Reply by tamlok** [reply]
Fixed: https://github.com/vnotex/vnote/commit/44b525104520070cb0f66d522d1a9cd719875f85

**Comment by minli04g** [comment]
解决了， 感谢。
